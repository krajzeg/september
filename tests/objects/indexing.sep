object := [[ apples: "crunchy", oranges: "juicy" ]]

print("Direct access via a string index:", object["apples"], "apples.")

fruit_name := "oranges"
print("Indirect access with a variable name:", object[fruit_name], "oranges.")

fruit_name = "apples"
object[fruit_name] = "rotten"
print("After modifying the values through indexing, we have:", object.apples, "apples.")
