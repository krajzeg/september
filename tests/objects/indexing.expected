Direct access via a string index: crunchy apples.
Indirect access with a variable name: juicy oranges.
After modifying the values through indexing, we have: rotten apples.
